Puppetfile

```diff
@@ -9,7 +9,7 @@ mod 'puppetlabs/yumrepo_core', '1.0.0'
 
 # Modules required to get a tomcat server up and running
 mod 'puppetlabs/tomcat', '2.5.0'
-mod 'puppetlabs/stdlib', '5.2.0'
+mod 'puppetlabs/stdlib', '4.25.1'
```

Comment:
Why is this version being changed? Should there be a comment with an explanation for this version being specified?

Reply:
I have added a comment for the restriction @billclaytor